nautobot/extras/forms.py

```diff
@@ -296,7 +298,7 @@ class Meta:
 class ComputedFieldFilterForm(BootstrapMixin, forms.Form):
     model = ComputedField
     q = forms.CharField(required=False, label="Search")
-    content_type = forms.ModelChoiceField(
+    content_type = CSVContentTypeField(
```

Comment:
These field changes for `ComputedFieldFilterForm` were required because the filtering *was not actually even working for this field*.